app/scripts/views/mixins/password-mixin.js

```diff
@@ -124,6 +130,16 @@ define(function (require, exports, module) {
       }
     },
 
+    setVisibilityShowPassword (el) {
+      const $passwordField = $(el);
+      const $showPasswordLabel = $passwordField.siblings('.show-password-label');
+      if ($passwordField.val().length !== 0) {
```

Comment:
In issue: "Simply make the button appear, without any special transition, when characters are entered"